recipes/pyvpt2/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pyvpt2" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyvpt2-{{ version }}.tar.gz
+  sha256: 431206261a18a00f925133259bb0ba28bd799b8f4c9ddb047d16691d50ec8c58
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools >=40.6.0
+    - wheel
+    - setuptools-git-versioning
+    - pip
+  run:
+    - python >=3.8
+    - pydantic <2
+    - numpy
+    - qcelemental
+    - qcengine
+    - psi4 >=1.8
+
+test:
+  imports:
+    - pyvpt2
+  commands:
+    - pip check
```

Comment:
if there's any quick script akin to https://github.com/conda-forge/libefp-feedstock/blob/main/recipe/test_pylibefp.py , https://github.com/conda-forge/libefp-feedstock/blob/main/recipe/meta.yaml#L122 , https://github.com/conda-forge/libefp-feedstock/blob/main/recipe/meta.yaml#L135 you can set up to run, it'll make the recipe more robust.

Reply:
I'm not sure that I have a quick test that could be run, but I did add the version checking.